django/db/migrations/graph.py

```diff
@@ -143,7 +143,7 @@ def add_dummy_node(self, key, origin, error_message):
     def add_dependency(self, migration, child, parent, skip_validation=False):
         """
         This may create dummy nodes if they don't yet exist. If
-        `skip_validation=True`, validate_consistency() should be called
```

Comment:
The existing text looks correct to me. If validation is skipped (skip_validation=True), the calling code should call validate_consistency() to do that instead of having this method do it.